Lib/test/test_os.py

```diff
@@ -1091,6 +1094,26 @@ def walk(self, top, **kwargs):
             bdirs[:] = list(map(os.fsencode, dirs))
             bfiles[:] = list(map(os.fsencode, files))
 
+@unittest.skipUnless(hasattr(os, 'fwalk'), "Test needs os.fwalk()")
+class BytesFwalkTests(FwalkTests):
+    """Tests for os.walk() with bytes."""
+    def setUp(self):
+        super().setUp()
+        self.stack = contextlib.ExitStack()
```

Comment:
It is not needed. This is just a copy from BytesWalkTests. But in BytesWalkTests it is not needed either. It was used in the past for silencing deprecation warnings about using Windows bytes API.